cybench/models/trend_models.py

```diff
@@ -114,48 +113,55 @@ def _predict_trend(self, test_data: Iterable):
 
```

Comment:
should we add an extra check that the test_year is removed from the training data?

Reply:
Added